recipe/meta.yaml

```diff
@@ -28,8 +28,9 @@ build:
     # With conda-forge/nvcc-feedstock#58, CUDA_PATH is set correctly
     - export NVCC=$(which nvcc)                                                  # [linux]
     - echo "nvcc is $NVCC, CUDA path is $CUDA_PATH"                              # [linux]
-    - for /f "tokens=* usebackq" %%f in (`where nvcc`) do (set "dummy=%%f" && call set "NVCC=%%dummy:\=\\%%")  # [win]
-    - echo "nvcc is %NVCC%, CUDA path is %CUDA_PATH%"                                                          # [win]
+    - for /f "tokens=* usebackq" %%f in (`where nvcc`) do (set "dummy=%%f" && call set "NVCC=%%dummy:\=\\%%")   # [win]
+    - for /f "tokens=* usebackq" %%f in (`where cl.exe`) do (set "dummy=%%f" && call set "CXX=%%dummy:\=\\%%")  # [win]
```

Comment:
@kmaehashi @jakirkham I added this like to expand conda-forge's `CXX` from "cl.exe" to the full path of cl.exe, but this error does not go away:
```
  'cl.exe' is not recognized as an internal or external command,
  operable program or batch file.
```
I am clueless as the last build went just fine and this PR, in particular, is simply a rebuild without anything changed (I think). I asked on conda-forge's gittter channel but no useful response yet. Any chance you guys have some ideas?

Reply:
cc @conda-forge/cupy

Reply:
https://github.com/conda-forge/nvcc-feedstock/pull/88